### ISSUE TITLE
Check for support for Django 4.0

### ISSUE THREAD

**Issue by philgyford** [philgyford/django-ditto#223]

**Comment by philgyford** [comment]
Now waiting for django-taggit to support Django 4.0 and python 3.10

**Comment by garrettc** [comment]
Looks like django-taggit 2.10 now has support for Django 4 and Python 3.10. It was something that [the Wagtail team were waiting for too](https://github.com/jazzband/django-taggit/issues/786).

**Comment by philgyford** [comment]
Yes, I saw that! Phew. I'll try and update at some point in the next few days.
